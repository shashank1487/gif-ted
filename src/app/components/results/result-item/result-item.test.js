import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure, mount } from "enzyme";
import ResultItem from "./result-item";

configure({ adapter: new Adapter() });
describe("The result item component", () => {
  let app;
  let images = {
    original: {
      url: "https://media3.giphy.com/media/pPhyAv5t9V8djyRFJH/giphy.gif"
    },
    "480w_still": {
      url: "https://media1.giphy.com/media/pPhyAv5t9V8djyRFJH/480w_s.jpg"
    }
  };

  beforeEach(() => {
    app = shallow(<ResultItem id="pPhyAv5t9V8djyRFJH" images={images} />);
  });

  it("should render correctly", () => {
    expect(app).toMatchSnapshot();
  });

  it("should render GifWrapper component", () => {
    expect(app.find('GifWrapper').exists()).toBeTruthy()    
  });
});
