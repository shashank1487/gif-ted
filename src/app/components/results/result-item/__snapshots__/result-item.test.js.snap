// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The result item component should render correctly 1`] = `ShallowWrapper {}`;
